nemo/backends/pytorch/common/other.py

```diff
@@ -1,12 +1,7 @@
 # Copyright (c) 2019 NVIDIA Corporation
 """Core PyTorch-base Neural Modules"""
 __all__ = [
-    'SimpleCombiner',
-    'ArgMaxSimple',
-    'TableLookUp',
-    'TableLookUp2',
     'SequenceEmbedding',
-    'SequenceProjection',
```

Comment:
Any reason why you have removed those modules?

Asking, as e.g. yesterday I was fixing one tutorial using the SimpleCombiner... In that case, shouldn't the associated tutorial be removed as well?

Besides, if those are deprecated, shouldn't we want to follow the procedure: deprecate for that revision, remove in the next one?

Reply:
I searched for references - and they were never used in code. They are also pretty poorly written/documented.